tools/cldr-apps/js/src/esm/cldrSideways.js

```diff
@@ -0,0 +1,212 @@
+/*
+ * cldrSideways: encapsulate Survey Tool code for the "sideways" menu,
+ * which enables switching and comparing between a set of related locales,
+ * such as: aa = Afar, aa_DJ = Afar (Djibouti), and aa_ER = Afar (Eritrea)
+ */
+import * as cldrDom from "./cldrDom.js";
+import * as cldrEvent from "./cldrEvent.js";
+import * as cldrLoad from "./cldrLoad.js";
+import * as cldrStatus from "./cldrStatus.js";
+import * as cldrSurvey from "./cldrSurvey.js";
+import * as cldrText from "./cldrText.js";
+
+/**
+ * Array storing all only-1 sublocale
+ */
+let oneLocales = [];
+
+/**
+ * Timeout for showing sideways view
+ */
+let sidewaysShowTimeout = -1;
+
+function loadMenu(frag, tr) {
+  if (oneLocales[cldrStatus.getCurrentLocale()]) {
+    return;
+  }
+  const sidewaysControl = cldrDom.createChunk(
```

Comment:
the remainder of this function is cut-pasted from cldrForumPanel.js without changes